### PR TITLE
Update free-programming-books-hu.md

### PR DESCRIPTION
**Pull request by yashsvi-30**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-hu.md [hunk 1]
```diff
@@ -36,6 +37,8 @@
 
 * [Arduino programozási kézikönyv](http://avr.tavir.hu) - Brian W. Evans írása alapján fordította, kiegészítette és frissítette Cseh Róbert (PDF - regisztráció szükséges)
 
+###Bootstrap
```
Comment (SethiPandi) [inline free-programming-books-hu.md:40]: Please add a line break below here.

File: free-programming-books-hu.md [hunk 2]
```diff
@@ -36,6 +37,8 @@
...
+* [#bootstrap free course](https://www.udemy.com/course/bootstrap-4/)-Brad Hussey(Online learning course- free of cost)
```
Comment (SethiPandi) [inline free-programming-books-hu.md:41]: Please remove the # symbol, write the title in title case, and give it a more useful name.

For example:

```
* [Bootstrap 4 Quick Start: Code Modern Responsive Websites](https://www.udemy.com/course/bootstrap-4/) - Brad Hussey
```

File: free-programming-books-hu.md [hunk 3]
```diff
@@ -36,6 +37,8 @@
...
```
Comment (SethiPandi) [inline free-programming-books-hu.md:42]: There should be two line breaks here, not one.

**Review by SethiPandi — COMMENTED** [review]
It's also worth noting this is a course, not a book.
Could you please move this to the courses document. It doesn't belong with the free books

File: free-programming-books-hu.md (hunk 1 above)
Comment (SethiPandi) [reply free-programming-books-hu.md:40]: Didn't notice until now, also you need to add space between `###` and `Bootstrap`.

File: free-programming-books-hu.md [hunk 4]
```diff
@@ -37,6 +38,11 @@
 * [Arduino programozási kézikönyv](http://avr.tavir.hu) - Brian W. Evans írása alapján fordította, kiegészítette és frissítette Cseh Róbert (PDF - regisztráció szükséges)
 
 
+### Bootstarp
+
+* [Bootstrap learning book](Sams Teach Yourself Bootstrap in 24 Hours)
```
Comment (SethiPandi) [inline free-programming-books-hu.md:43]: This doesn't include a link to a book whatsoever anymore.

Comment (SethiPandi) [reply free-programming-books-hu.md:43]: Also, you've spelled Bootstrap wrong now.

**Comment by eshellman** [comment]
not an authorized distribution

**Pull request closed without merging**